content/news/vcmi-0.100-release.md

```diff
@@ -0,0 +1,35 @@
++++
+categories = ["Releases", "Stable"]
+date = "2022-09-10"
+description = ""
+featured = "vcminewterrain.jpg"
+featuredalt = ""
+featuredpath = "date"
+linktitle = ""
+title = "VCMI 0.100 released"
+type = "post"
+forumurl = "https://forum.vcmi.eu/"
+
++++
+
+## We're happy to announce new release!
+
+Few years passed after we released VCMI 0.99 and now it's time for a new release.
+Tons of changes, improvements, and fixes have been introduces during that time, it's even hard to recall all of them.
+However we'd like to highlight several major changes.
+
+## Artificial intellengence
+You are welcome to try brand new AI algorithm called Nullkiller or you can play with classic VCAI computer player.
+Both algorithms became much smarter in comparison with last release, so just choose from the laucher settings whatever you like and enjoy.
+
+## Mods and new terrains
+VCMI opens broad opportunitites for modding. Since last release we significantly empowered mod system.
+Among various gameplay tools we introduced support of new terrains. It is a natural evolution of new towns as they need to have unique terrains, battlefields, mines and map objects. This feature required deep refactoring of game engine so we all saves you made before release 0.100 will not be compatible. So it's a good reason to make new ones!
+
+## Random Maps Generator
+RMG also has undergone huge changes. First of all, water modes are supported so all options from game menu work well.
+Of course supporting new terrains from mods is useless if you can't play with it. RMG has powerful algorithms to work with new terrains and placing new objects. Moreover, we improved generator in terms of aesthetics, obstacles placement and stability as well.
```

Comment:
Nit: this sentence goes as if it continues some previous idea. But the previous sentence is about water maps and it is not related to new terrains. it confused me and it took some time to get the entire paragraph clearly. Maybe, rewrite it a bit. As an example (replace two first sentences from L31 with this): 

As mods can introduce new terrains and objects, RNG got powerful algorithms to generate new terrain patches and to place new objects provided by mods.